jbpm-form-modeler-panels/jbpm-form-modeler-editor/jbpm-form-modeler-editor-client/src/main/java/org/jbpm/formModeler/editor/client/editors/FormModelerPanelPresenter.java

```diff
@@ -198,19 +204,13 @@ public IsWidget getWidget() {
     }
 
     protected void makeMenuBar() {
-        menus = menuBuilder
-                .addSave(versionRecordManager.newSaveMenuItem(new Command() {
-                    @Override
-                    public void execute() {
-                        onSave();
-                    }
-                }))
+        fileMenuBuilder
```

Comment:
@pefernan There was an issue with subclasses of ```KieEditor``` masking the "Menu Builder" which leads to menus not being shown for the editor. This change removes the masking.